src/EditorFeatures/TestUtilities/DocumentationComments/AbstractDocumentationCommentTests.cs

```diff
@@ -123,13 +123,15 @@ private static Action CreateInsertTextHandler(ITextView textView, string text)
         {
             using (var workspace = CreateTestWorkspace(initialMarkup))
             {
+                var globalOptions = workspace.ExportProvider.GetExportedValue<IEditorOptionsFactoryService>().GlobalOptions;
+                globalOptions.SetOptionValue(DefaultOptions.NewLineCharacterOptionId, newLine);
+
                 var testDocument = workspace.Documents.Single();
 
                 var options = workspace.Options;
 
                 options = options.WithChangedOption(FormattingOptions.UseTabs, testDocument.Project.Language, useTabs);
                 options = options.WithChangedOption(FeatureOnOffOptions.AutoXmlDocCommentGeneration, testDocument.Project.Language, autoGenerateXmlDocComments);
-                options = options.WithChangedOption(FormattingOptions.NewLine, testDocument.Project.Language, newLine);
```

Comment:
📝 It doesn't seem like an improvement that explicitly set options are now getting ignored.

Reply:
yes, we need to find a way to read these only as a fallback